rides/templates/mail/opening.html

```diff
@@ -0,0 +1,13 @@
+{% extends "extend/email.html" %}
+
+{% block body %}
+    <div class="container main">
+        <h2>Hello {{ rider }},</h2>
+        <h3>There is an available ride for {{ event }}!</h3>
+        <p>Driver: {{ driver }}</p>
+        <p>If you wish to join this ride, please go to <a href={{ url }}>{{ url }}</a></p>
+        <p>Ensure that you were added to the ride, if not please hit the "Join Ride" button!</p>
+        <p>IF YOU USE A CSH LOGIN, YOU CAN HAVE RIDEBOARD MESSAGE YOU ON SLACK INSTEAD OF EMAIL!</p>
```

Comment:
Maybe only include this bit if they're a CSH user?

Reply:
https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/94e6d380f1db3a213203cf09c1bae9eae9edb71f